src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machine = new Machine[] {new Truck(), new Bulldozer(), new Excavator()};
+        for (Machine m : machine) {
+            m.doWork();
+            m.stopWork();
+        }
```

Comment:
I'd recommend avoiding the shortened names, it makes the code less readable
```suggestion
        for (Machine machine : machine) {
            machine.doWork();
            machine.stopWork();
        }
```

Reply:
thx, fixed.